x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.tsx

```diff
@@ -131,467 +137,469 @@ export function LayerPanel(
   const columnLabelMap = layerDatasource.uniqueLabels(layerDatasourceConfigProps.state);
   return (
     <ChildDragDropProvider {...dragDropContext}>
-      <EuiPanel
-        data-test-subj={`lns-layerPanel-${index}`}
-        className="lnsLayerPanel"
-        paddingSize="s"
-      >
-        <EuiFlexGroup gutterSize="s" alignItems="flexStart" responsive={false}>
-          <EuiFlexItem grow={false} className="lnsLayerPanel__settingsFlexItem">
-            <LayerSettings
-              layerId={layerId}
-              layerConfigProps={{
-                ...layerVisualizationConfigProps,
-                setState: props.updateVisualization,
-              }}
-              activeVisualization={activeVisualization}
-            />
-          </EuiFlexItem>
-
-          {layerDatasource && (
-            <EuiFlexItem className="lnsLayerPanel__sourceFlexItem">
-              <NativeRenderer
-                render={layerDatasource.renderLayerPanel}
-                nativeProps={{
-                  layerId,
-                  state: layerDatasourceState,
-                  activeData: props.framePublicAPI.activeData,
-                  setState: (updater: unknown) => {
-                    const newState =
-                      typeof updater === 'function' ? updater(layerDatasourceState) : updater;
-                    // Look for removed columns
-                    const nextPublicAPI = layerDatasource.getPublicAPI({
-                      state: newState,
-                      layerId,
-                    });
-                    const nextTable = new Set(
-                      nextPublicAPI.getTableSpec().map(({ columnId }) => columnId)
-                    );
-                    const removed = datasourcePublicAPI
-                      .getTableSpec()
-                      .map(({ columnId }) => columnId)
-                      .filter((columnId) => !nextTable.has(columnId));
-                    let nextVisState = props.visualizationState;
-                    removed.forEach((columnId) => {
-                      nextVisState = activeVisualization.removeDimension({
-                        layerId,
-                        columnId,
-                        prevState: nextVisState,
-                      });
-                    });
-
-                    props.updateAll(datasourceId, newState, nextVisState);
-                  },
+      <section tabIndex={-1} ref={setLayerRefMemoized}>
```

Comment:
I am surprised that git analyzed this file this way. Line 140 is the only changed like in the whole `return` statement.

Reply:
Reviewed with this flag on and it's only 2 lines change:
<img width="234" alt="Screenshot 2020-12-15 at 17 06 17" src="https://user-images.githubusercontent.com/924948/102240212-e4b76300-3ef7-11eb-8976-e5c5dfb93c8c.png">

Reply:
Nice!!! Thanks for the tip, I forgot about this!